### PR TITLE
[Delivers #158791514] Logout API Endpoint

### PR DESCRIPTION
**Pull request by JoshuaOndieki**
**What does this PR do?**
Adds logout endpoint to the API

**Description of Task to be completed**
A user should be able to log out

POST `/api/v2/auth/login`

**Tasks achieved in this PR**
- [x] POST `/api/v2/auth/logout`

**How should one test this endpoint?**
- Clone this repo using `git clone https://github.com/JoshuaOndieki/ridemyway-api.git`
- Check the ReadMe for instructions on setting up the project. 
- `git checkout ft-logout-158791514` 
- Run `nosetests` to see the login tests pass. Alternatively, run `pytest`

**How can you help?**
Review my work and let me know any semantic, technical or syntax/standards errors I may have.
I will greatly appreciate any feature/change suggestions too :smiley: 

**Resources & Libs**
[This Pivotal Tracker board](https://www.pivotaltracker.com/n/projects/2179581) was used to plan and manage the tasks.

**Relevant PIvotal Tracker Stories?**
[#158791514](https://www.pivotaltracker.com/story/show/158791416)

### CODE REVIEW

File: ridemyway/tests/tests_v2/test_auth/test_signup.py [hunk 1]
```diff
@@ -23,7 +23,6 @@ class TestSignUp(V2BaseTest):
     """
 
     def test_driver_can_signup_successfully(self):
-        print(VALID_DRIVER)
         self.response = self.client().post(SIGNUP, data=VALID_DRIVER)
         self.assertEqual(self.response.status_code, 201,
```
Comment (ghost) [inline ridemyway/tests/tests_v2/test_auth/test_signup.py:27]: Add edge cases to your tests e.g if name filed is empty

Comment (JoshuaOndieki) [reply ridemyway/tests/tests_v2/test_auth/test_signup.py:27]: I've actually covered that case for signup. Thanks for the review as it has helped me notice that case is not covered in the login endpoint.

Comment (JoshuaOndieki) [reply ridemyway/tests/tests_v2/test_auth/test_signup.py:27]: I'll make the suggested changes in the login PR #30

**Comment by kwameasiago** [comment]
I see everything is ok
